src/metric_model_reporter.cc

```diff
@@ -341,9 +358,9 @@ MetricModelReporter::IncrementCounter(const std::string& name, double value)
     // No counter metric exists with this name
     return;
   }
-
+  
   auto counter = iter->second;
-  if (!counter) {
+  if (!counter){
```

Comment:
You should setup the pre-commit hooks like `pre-commit install` and `pre-commit run` to auto-run the linter/formatters when you commit the code.